build-tools/packages/build-cli/package.json

```diff
@@ -92,9 +92,11 @@
     "oclif": "^3",
     "semver": "^7.3.7",
     "semver-utils": "^1.1.4",
+    "simple-git": "^3.14.1",
     "sort-json": "^2.0.1",
```

Comment:
why are you using simple-git here? can gitRepo.mergeBase or gitRepo.revList be used instead?

Reply:
Good catch -- this is intentional. One of the minor security issues the build tools have is that they exec a lot of stuff directly. This command was scoped enough that I felt like trying to avoid the security warnings in this PR. That was successful -- but there are still calls to `raw` which may not be sanitized enough.